doc/source/contributing/index.md

```diff
@@ -1,50 +1,60 @@
-* Contributing or improving the {ref}`documentation <homapage>` (`docs`) or examples (`arviz/examples`).
-* Help users submitting issues by making sure their issue is clear and/or providing temporal
-  workarounds for them. Check the details at {ref}`issue_triaging`
-* Review Pull Requests ensuring contributions are well tested and documented, that documentation
-  renders correctly and has no typos or mistakes. Check the details at {ref}`review_prs`
-* Adding new or improved functionality to the existing codebase.
+### Translate ArviZ website
+You can [translate](https://python.arviz.org/en/latest/contributing/translate.html#translate) our website to languages other than english. 
 
-Further information on how to contribute to ArviZ can be found on these specific pages:
+### Review PRs
+[Review pull requests](https://arviz-devs.github.io/arviz/contributing/review_prs.html#review-prs) to ensure that the contributions are well tested and documented. For example, you can check if the documentation renders correctly and has no typos or mistakes. 
+
+### Manage issues
+Helping to manage or triage the open issues can be a great contribution and a great opportunity to learn about the various areas of the project. You can [triage existing issues](https://arviz-devs.github.io/arviz/contributing/issue_triaging.html#issue-triaging) to make them clear and/or provide temporary workarounds for them. 
+
+### Write documentation
+You can contribute to ArviZ’s documentation by either creating new content or editing existing content. For instance, you can add new ArviZ examples. To get involved with the documentation:
+1. Familiarize yourself with the [documentation content structure](https://arviz-devs.github.io/arviz/contributing/content_structure.html) and the [tool chain](https://arviz-devs.github.io/arviz/contributing/doc_toolchain.html) involved in creating the website.
+2. Understand the basic workflow for opening a [pull request](https://arviz-devs.github.io/arviz/contributing/pr_tutorial.html) and [reviewing changes](https://arviz-devs.github.io/arviz/contributing/review_prs.html).
+3. Work on your content.
+4. [Build](https://arviz-devs.github.io/arviz/contributing/sphinx_doc_build.html) your documentation and preview the doc changes. 
+   
+### Support outreach initiatives
+Support ArviZ [outreach initiatives](https://arviz-devs.github.io/arviz/contributing/outreach.html) such as writing blog posts, case studies,  getting people to use ArviZ at your company or university, etc.
+
+### Make changes in the code
+Fix outstanding issues (bugs) in the existing codebase. The issue can range from low-level software bugs to high-level design problems. You can also add new features to the codebase or improve the existing functionality. To fix bugs or add new features, do the following:
+1. Familiarize yourself with the [guidelines](https://arviz-devs.github.io/arviz/contributing/contributing_prs.html#steps-before-starting-work) before starting your work.
```

Comment:
```suggestion
1. Familiarize yourself with the {ref}`guidelines <steps_before_working>` before starting your work.
```

needs to have `(steps_before_working)=` added right above the `
## Steps before starting work` line in `contributing_prs.md`